schedule.json

```diff
@@ -113,5 +113,29 @@
     "maintenance": "2024-10-22",
     "end": "2026-04-30",
     "codename": ""
+  },
+  "v21": {
+    "start": "2023-10-17",
+    "maintenance": "2024-04-01",
+    "end": "2024-06-01"
+  },
+  "v22": {
+    "start": "2024-04-23",
```

Comment:
I think we usually go for the third Tuesday in the month, although I'm not sure that's written down as policy anywhere.

Reply:
Does it matter? I counted from the other side (second to last Tuesday in the month)